### ISSUE TITLE
[accessibility]: hide the mobile menu toggle from the accessiblity tree for Desktop users. 

### ISSUE THREAD

**Issue by nisarhassan12** [gitpod-io/website#1819]
Currently while using a Screen Reader if you tab through the nav links on Desktop the menu toggle button label is read which should not be the case.

**Comment by levi-bassey** [comment]
Hi @nisarhassan12, can I try resolving this issue?

**Comment by nisarhassan12** [comment]
@levi-bassey Thanks ❤️ Yeah sure.

**Comment by levi-bassey** [comment]
@nisarhassan12, just to clarify your'e referring to this toggle right?
![image](https://user-images.githubusercontent.com/43778081/160256427-e903bcba-75bd-498e-adae-0cff936d1a89.png)

**Comment by levi-bassey** [comment]
So the problem is that the buttons label gets read on desktop when it should be hidden right? I'm asking because I tried it but I can't seem to reproduce the issue. The button is hidden and my screen reader(chrome vox) doesn't detect the toggle on desktop. Please correct me if I'm missing something, thanks.

**Comment by nisarhassan12** [comment]
> @nisarhassan12, just to clarify your'e referring to this toggle right?

Yep, @levi-bassey 

> my screen reader(chrome vox) doesn't detect the toggle on desktop. Please correct me if I'm missing something, thanks.

I tested on Voice Over on Mac OS and it seems to read that.

**Comment by nisarhassan12** [comment]
Thanks, @levi-bassey for looking into it. Since you don't have a way to reproduce this so I'll take over this.

**Comment by nisarhassan12** [comment]
Not sure what happened and it is quite odd that now I'm also no longer able to hear the toggle on Mac via Voice Over 😄 Will close this issue please re-open in case anyone is able to reproduce this.
